This patch enable building all linux supported LCL widget sets.

Index: lazarus/lcl/Makefile.fpc
===================================================================
--- lazarus.orig/lcl/Makefile.fpc
+++ lazarus/lcl/Makefile.fpc
@@ -156,6 +156,7 @@ cleanintf:
 	$(MAKE) -C interfaces clean
 
 all: lclbase compiled intf
+	for WS in gtk2 nogui qt5 ; do ${MAKE} intf LCL_PLATFORM=$${WS} ; done
 
 cleanlaz: cleanall
 cleanall: clean
